.gitignore

```diff
@@ -10,3 +10,33 @@ packages/*/*.js
 packages/*/bin
 !packages/*/.eslintrc.js
 *.log
+
+# Built style type definitions
+**/next-env.d.ts
+**/.next
```

Comment:
Why these?

Reply:
Hmm - I'm not sure why either. I'll remove them